WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabView.xaml

```diff
@@ -23,6 +23,11 @@
       <RowDefinition Height="3*" MinHeight="200" />
     </Grid.RowDefinitions>
     <Grid Classes="content" Grid.Row="0" Margin="0 0 0 10">
+      <Grid.Styles>
+        <Style Selector="ListBoxItem">
+          <Setter Property="Padding" Value="0 0 5 0" />
```

Comment:
Can it be set by here somewhere? So you have to only change CoinList?
https://github.com/molnard/WalletWasabi/blob/bde56e10a2f1a5e3f7dc4274905290d66c838c2d/WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml#L88

Reply:
unfortunately we only want this padding to apply to the pages that have CoinListView with Expanders directly in the items.